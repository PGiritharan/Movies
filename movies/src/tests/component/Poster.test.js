import React from 'react';
import { shallow } from 'enzyme';
import Poster from '../../component/poster';
import movie from '../fixures/movies';
import toJson from 'enzyme-to-json';

test('should render Poster correctly', () => {
  const wrapper = shallow(<Poster movie={movie}/>);
  expect(toJson(wrapper)).toMatchSnapshot();
});
