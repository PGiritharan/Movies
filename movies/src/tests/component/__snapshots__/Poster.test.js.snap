// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Poster correctly 1`] = `
<div
  className="poster"
>
  <img
    alt=""
    className="poster-img"
    loading="lazy"
    onClick={[Function]}
    src="testurl.jpg"
  />
</div>
`;
